config/webpack.config.js

```diff
@@ -24,53 +47,56 @@ module.exports = {
 		loaders: [
 			{
 				test: /\.js$/,
-				loader: require.resolve('babel-loader'),
-				include: [
-					/bower_components/,
-					path.resolve(__dirname, 'client'),
-					path.resolve(__dirname, 'config'),
-					path.resolve(__dirname, 'shared')
-				].concat((config.assets.includes || []).map(path => new RegExp(path))),
+				loader: 'babel',
+				include: (() => {
+					const includes = [
+						/bower_components/,
+						path.resolve(__dirname, './client'),
+						path.resolve(__dirname, './config'),
+						path.resolve(__dirname, './shared')
+					];
+
+					if (config.assets.includes) {
+						config.assets.includes.forEach(path => {
+							includes.push(new RegExp(path));
+						});
+					}
+
+					return includes;
```

Comment:
This is quite a bit longer but I thought it was worth expanding so it's easier to see what's happening at a glance